dev/02_data_transforms.ipynb

```diff
@@ -535,13 +535,25 @@
    "cell_type": "markdown",
```

Comment:
<p>In "handling of tuples", there is a typo to transform. You should also say you have to set <code>as_item=True</code> to opt out of this behavior and expand a little bit more about <code>as_item_force</code> saying that setting this flag will always override <code>as_item</code> (for transforms that can only be applied at a tuple level such as <code>PointScaler</code>).</p><p> </p><p>In "Reversability", your example is wrong: a <code>TensorImage</code> is something we can show and the transform that converts an image to a tensor is not reversed. A better example would be a category converted to the corresponding index that you need to reverse for showing purposes.</p><p> </p><p>In "Filtering based on the dataset type", you should say used only on a specific <code>DataSource</code> subset (the datasource has a training and validation set).</p><p> </p><p>In "appending a new behavior..." I think you should point out that it's for new types of objects with an annotation.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>